docs/settings.rst

```diff
@@ -0,0 +1,189 @@
+.. _settings:
```

Comment:
Added specific section for the settings (extracted from `usage.rst`). Multiple errors and improvements have been performed.